CHANGELOG.md

```diff
@@ -1,5 +1,301 @@
+* Adds a new RoutingFilter, MaximumLagFilter that looks at offset lags ([#4468](https://github.com/confluentinc/ksql/pull/4468)) ([fd7c1a5](https://github.com/confluentinc/ksql/commit/fd7c1a5))
+* Adds lag reporting and API for use in lag aware routing as described in KLIP 12 ([#4392](https://github.com/confluentinc/ksql/pull/4392)) ([cb9ae29](https://github.com/confluentinc/ksql/commit/cb9ae29))
+* better error message when transaction to command topic fails to initialize by timeout ([#4486](https://github.com/confluentinc/ksql/pull/4486)) ([a5fed3b](https://github.com/confluentinc/ksql/commit/a5fed3b))
+* expression support in JOINs ([#4278](https://github.com/confluentinc/ksql/pull/4278)) ([2d0bfe8](https://github.com/confluentinc/ksql/commit/2d0bfe8))
+* hide internal/system topics from SHOW TOPICS ([#4322](https://github.com/confluentinc/ksql/pull/4322)) ([075fed3](https://github.com/confluentinc/ksql/commit/075fed3))
+* Implement pull query routing to standbys if active is down ([#4398](https://github.com/confluentinc/ksql/pull/4398)) ([ace23b1](https://github.com/confluentinc/ksql/commit/ace23b1))
+* Implementation of heartbeat mechanism as part of KLIP-12 ([#4173](https://github.com/confluentinc/ksql/pull/4173)) ([37c1eaa](https://github.com/confluentinc/ksql/commit/37c1eaa))
+* improve the semantics around decimal/floating point literals ([#4335](https://github.com/confluentinc/ksql/pull/4335)) ([d15026f](https://github.com/confluentinc/ksql/commit/d15026f))
+* native map/array constructors ([#4232](https://github.com/confluentinc/ksql/pull/4232)) ([3ecfaad](https://github.com/confluentinc/ksql/commit/3ecfaad))
+* remove WindowStart() and WindowEnd() UDAFs ([#4459](https://github.com/confluentinc/ksql/pull/4459)) ([eda2e34](https://github.com/confluentinc/ksql/commit/eda2e34))
+* support implicit casting in UDFs ([#4406](https://github.com/confluentinc/ksql/pull/4406)) ([6fc4f72](https://github.com/confluentinc/ksql/commit/6fc4f72))
+* Support JSON format for responses and other refactorings ([#4376](https://github.com/confluentinc/ksql/pull/4376)) ([1ecaeee](https://github.com/confluentinc/ksql/commit/1ecaeee))
+* support Protobuf in ksqlDB ([#4469](https://github.com/confluentinc/ksql/pull/4469)) ([a77cebe](https://github.com/confluentinc/ksql/commit/a77cebe))
+* ask for password if -p is not provided ([#4153](https://github.com/confluentinc/ksql/pull/4153)) ([7a83bbf](https://github.com/confluentinc/ksql/commit/7a83bbf))
+* add a KSQL cache for Kafka authorization validator ([#4186](https://github.com/confluentinc/ksql/pull/4186)) ([58ed39c](https://github.com/confluentinc/ksql/commit/58ed39c))
+* add config to make error messages configurable ([#4121](https://github.com/confluentinc/ksql/pull/4121)) ([cedf47e](https://github.com/confluentinc/ksql/commit/cedf47e))
```

Comment:
I think this can either be removed, or reworded to inform users how this relates to the UX from their perspective.

Reply:
Reworded to "make (certain types) of server error messages configurable"